modules/preview-react/pill/lib/Pill.tsx

```diff
@@ -0,0 +1,218 @@
+import React from 'react';
+
+import {
+  createContainer,
+  focusRing,
+  mouseFocusBehavior,
+  styled,
+  StyledType,
+} from '@workday/canvas-kit-react/common';
+
+import {usePillModel} from './usePillModel';
+
+import {PillIcon} from './PillIcon';
+import {PillIconButton} from './PillIconButton';
+import {PillCount} from './PillCount';
+import {PillAvatar} from './PillAvatar';
+import {borderRadius, colors, space, type} from '@workday/canvas-kit-react/tokens';
+import {BoxProps, boxStyleFn, HStack, Stack} from '@workday/canvas-kit-react/layout';
+import {CSSObject} from '@emotion/react';
+import {PillLabel} from './PillLabel';
+import {BaseButton} from '@workday/canvas-kit-react/button';
```

Comment:
Let's organize these a bit:

```suggestion
import {CSSObject} from '@emotion/react';

import {BaseButton} from '@workday/canvas-kit-react/button';
import {
  createContainer,
  focusRing,
  mouseFocusBehavior,
  styled,
  StyledType,
} from '@workday/canvas-kit-react/common';
import {BoxProps, boxStyleFn, HStack, Stack} from '@workday/canvas-kit-react/layout';
import {borderRadius, colors, space, type} from '@workday/canvas-kit-react/tokens';

import {usePillModel} from './usePillModel';

import {PillIcon} from './PillIcon';
import {PillIconButton} from './PillIconButton';
import {PillCount} from './PillCount';
import {PillAvatar} from './PillAvatar';
import {PillLabel} from './PillLabel';
```